### ISSUE TITLE
Pretrained model gives low validation accuracy

### ISSUE THREAD

**Issue by yunhuiguo** [srebuffi/residual_adapters#1]
Hi,

I download the pretrained ResNet-26 and use the code of finetuning to test the validation accuracy on ImageNet, however it gives roughly 0% accuracy.

**Comment by AlenUbuntu** [comment]
me too. The accuracy is pretty low.
